apprentice_teacher.rb

```diff
@@ -1,53 +1,23 @@
-class ApprenticeTeacher
-  attr_reader :age, :salary, :phase, :target_raise
-  attr_accessor :name
+require_relative 'Teacher'
+
+class ApprenticeTeacher < Teacher
+  include Teachable
 
   def initialize(options={})
-    @age = options.fetch(:age, 0)
-    @name = options.fetch(:name, "")
+    super
     @target_raise = 800
-    @phase = 3
-  end
-
-  def offer_high_five
-    "High five!"
+    @benchmark_rating = 80
```

Comment:
This is answered in a comment on teacher now, but essentially it's a question that depends on how we evaluate teachers. If these are the same for every teacher at this level, then constants are the way to go. 

Honestly, I struggled with constants quite a lot on Monday night and wouldn't have used them in the timed test even if I'd realized they were the right choice for this. Got it now though 😄